include/nccl_ofi_memcheck_builtin.h

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright 2014-2023 Amazon.com, Inc. or its affiliates.  All Rights Reserved.
+ */
+
+#ifndef NCCL_OFI_MEMCHECK_BUILTIN_H
```

Comment:
why does this exist?  what does the builtin memcheck do?

Reply:
This header file is included in case no memory access checker is enabled. In this case it provides empty function definitions.

Reply:
Sorry, that was a request for documentation.

Reply:
I see.